_pages/en_US/restoring-nand.md

```diff
@@ -49,7 +49,17 @@ It is very important to test that your NAND backup is working before attempting
 8. Click `OK`
 9. Launch any Nintendo DS ROM (`.nds` file)
 
-If no$gba loads the DSi menu, then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+If no$gba loads the DSi menu (or the Unlaunch Filemenu), then continue to the next section. If it shows any kind of error ***do not flash that backup***!
+
+## Uninstalling Unlaunch from your NAND backup (optional)
+Follow this if you dumped your NAND backup after you installed Unlaunch and you would like to uninstall Unlaunch from your system.
```

Comment:
```suggestion
Follow this if you dumped your NAND backup after you installed Unlaunch and you would like to uninstall Unlaunch from your system. If you are not trying to install Unlaunch you do **not** need to do this section.
```

I think it'd be good to explicitly say not to do this if you're not trying to uninstall Unlaunch, since people may ignore the (optional) in the title.

Reply:
I think it's important to make it as clear as possible, and allow people to identify with it more easily.